composer.json

```diff
@@ -94,6 +94,7 @@
         "symfony/security-csrf": "^4.4",
         "symfony/security-guard": "^4.4",
         "symfony/security-http": "^4.4",
+        "symfony/serializer": "^5.0",
```

Comment:
I would rather go with "symfony/serializer" 4.4, as this is the official supported version for Symfony packages. We do not have support for 5.0 for any library, so we can migrate all these libraries together later.

Reply:
👍 but then we need to change the denormalizers implementation 💃